src/include/network/lan/complex.rb

```diff
@@ -285,6 +285,25 @@ def enableDisableButtons
       nil
     end
 
+    # Updates the udev rule of the current Lan Item, adding the dev_port and
+    # using the bus_id instead ot the mac address.
+    def use_udev_rule_for_bonding!
```

Comment:
well, ```use_udev_rule_for_bonding``` sounds like when udev rule is used for bonding only. I'd use something like ```create_...``` or ```update_...```

also you should either document return value or return nil explicitly (I'd prefer the later one)